doc/classes/Node3D.xml

```diff
@@ -139,91 +142,95 @@
 			<param index="2" name="up" type="Vector3" default="Vector3(0, 1, 0)" />
 			<param index="3" name="use_model_front" type="bool" default="false" />
 			<description>
-				Moves the node to the specified [param position], and then rotates the node to point toward the [param target] as per [method look_at]. Operations take place in global space.
+				Moves the node to the specified [param position], and then rotates the node to point toward the [param target] as per [method look_at]. This operation is calculated in global space (relative to the world).
```

Comment:
You change this part from "Operations take place" to "This operation is calculated", but the one in "look_at" still says "Operations take place". We should pick one wording and be consistent.